index.bs

```diff
@@ -170,2193 +138,170 @@ The important commonality between them being that they offer some degree of spat
-  1. Let |pose| be a new {{XRViewerPose}} object.
-  1. [=Populate the pose=] of |session|'s [=XRSession/viewer reference space=] in |referenceSpace| at the time represented by |frame| into |pose|.
-  1. If |pose| is <code>null</code> return <code>null</code>.
-  1. Let |xrviews| be an empty [=/list=].
-  1. For each [=view=] |view| in the [=XRSession/viewer reference space/list of views=] on the[=XRSession/viewer reference space=] of {{XRFrame/session}}, perform the following steps:
-      1. Let |xrview| be a new {{XRView}} object.
-      1. Initialize |xrview|'s {{XRView/eye}} to |view|'s [=view/eye=]
-      1. Initialize |xrview|'s {{XRView/projectionMatrix}} to |view|'s [=view/projection matrix=]
-      1. Let |offset| be an {{XRRigidTransform}} equal to the [=view offset=] of |view|
-      1. Set |xrview|'s {{XRView/transform}} property to the result of [=multiply transforms|multiplying=] the {{XRViewerPose}}'s {{XRPose/transform}} by the |offset| transform
-      1. [=list/Append=] |xrview| to |xrviews|
-  1. Set |pose|'s {{XRViewerPose/views}} to |xrviews|
-  1. Return |pose|.
-
-</div>
-
-<div class="algorithm" data-algorithm="get-pose">
-
-When the <dfn method for="XRFrame">getPose(|space|, |baseSpace|)</dfn> method is invoked, the user agent MUST run the following steps:
-
-  1. Let |frame| be the target {{XRFrame}}
-  1. Let |pose| be a new {{XRPose}} object.
-  1. [=Populate the pose=] of |space| in |baseSpace| at the time represented by |frame| into |pose|.
-  1. Return |pose|.
-
-</div>
-
-Spaces {#spaces}
-======
-
-A core feature of the WebXR Device API is the ability to provide spatial tracking. Spaces are the interface that enable applications to reason about how tracked entities are spatially related to the user's physical environment and each other.
-
-XRSpace {#xrspace-interface}
--------
-
-An {{XRSpace}} represents a virtual coordinate system with an origin that corresponds to a physical location. Spatial data that is requested from the API or given to the API is always expressed in relation to a specific {{XRSpace}} at the time of a specific {{XRFrame}}. Numeric values such as pose positions are coordinates in that space relative to its origin. The interface is intentionally opaque.
-
-<pre class="idl">
-[SecureContext, Exposed=Window] interface XRSpace : EventTarget {
-
-};
-</pre>
-
-Each {{XRSpace}} has a <dfn for="XRSpace">session</dfn> which is set to the {{XRSession}} that created the {{XRSpace}}.
-
-Each {{XRSpace}} has a <dfn for="XRSpace">native origin</dfn> that is tracked by the [=XRSession/XR device=]'s underlying tracking system, and an <dfn for="XRSpace">effective origin</dfn>, which is the basis of the {{XRSpace}}'s <dfn for="XRSpace">coordinate system</dfn>. The transform from the effective space to the [=native origin=]'s space is defined by an <dfn for="XRSpace">origin offset</dfn>, which is an {{XRRigidTransform}} 
```

Comment:
Any particular reason why this specific paragraph is left in? Much of it reads as headset-specific, and we probably only want to list privacy concerns here that directly relate to the information exposed by the gamepad inputs. (Not saying that isn't fingerprintable, just that we'd want to express that risk more explicitly.)

Also, an issue for the main spec: The line "devices that are reported have no string identifiers" could be read as conflicting with the data surfaced by the `profiles` array.

Reply:
I wasn't finished yet ^_^; should be gone now